examples/hello-world/step-by-step/cifar10/sag/sag.ipynb

```diff
@@ -0,0 +1,285 @@
+    "\n",
+    "For detailed discussion how to convert training code into federated learning training code using Client API, you can also checked out the examples [here](https://github.com/NVIDIA/NVFlare/blob/main/examples/hello-world/ml-to-fl/README.md) and code \n",
+    "\n",
+    "The key changes are the following steps: \n",
+    "\n",
+    "```\n",
+    "    #  import nvflare client API\n",
+    "    import nvflare.client as flare\n",
+    "\n",
+    "    #  initializes NVFlare client API\n",
+    "    flare.init()\n",
+    "\n",
+    "    # gets FLModel from NVFlare\n",
+    "    input_model = flare.receive()\n",
+    "\n",
+    "    # loads model from NVFlare\n",
+    "    net.load_state_dict(input_model.params)\n",
+    "\n",
+    "    # evaluate on received model\n",
+    "    accuracy = evaluate(input_model.params)\n",
+    "    \n",
+    "    # construct trained FL model\n",
+    "    output_model = flare.FLModel(\n",
+    "        params=net.cpu().state_dict(),\n",
+    "        metrics={\"accuracy\": accuracy},\n",
+    "        meta={\"NUM_STEPS_CURRENT_ROUND\": steps},\n",
+    "    )\n",
+    "    \n",
+    "    # send model back to NVFlare\n",
+    "    flare.send(output_model)\n",
+    "```\n",
+    "\n",
+    "If you are using pytorch-lightning, the changes are much smaller, 1-line import , 1-line change applies to trainer, 1-line global model evaluation. see [cifar10_lightning_examples](https://github.com/NVIDIA/NVFlare/blob/main/examples/hello-world/ml-to-fl/codes/cifar10_lightning.py) \n",
+    "# Prepare Data\n",
+    "\n",
+    "Let's get the data first. Follow the instruction of cifar10, we can download the data with following scripts. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7e487166-1e47-47db-83cc-f482735b76a5",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "CIFAR10_ROOT = \"/tmp/nvflare/data/cifar10\"\n",
+    "! python ../data/download.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6667a9be-a911-4485-a3cb-cc2067852545",
+   "metadata": {},
+   "source": [
+    "## Job Folder and Configurations\n",
+    "\n",
+    "Now we need to setup the configurations for server and clients and constructure Job folder NVFLARE needed to run. We can do this using NVFLARE job CLI. You can study the [Job CLI tutorials](https://github.com/NVIDIA/NVFlare/blob/main/examples/tutorials/job_cli.ipynb) later with all the details. But for now, you can just use the following commands\n",
+    "\n",
+    "* First Let's explain the available job templates\n",
+    "\n",
+    "make sure we know where to find the job templates. If have already set NVFLARE_HOME=<NVFLARE git clone directory> then, you can skipt the folllowing step. \n"
```

Comment:
This sentence "* First Let's explain the available job templates\n" is not consistent with the paragraph above and below.